src/glossary.md

```diff
@@ -52,6 +59,11 @@ A variable is initialized if it has been assigned a value and hasn't since been
 moved from. All other lvalues are assumed to be initialized. Only unsafe Rust
 can create such an lvalue without initializing it.
 
+### Nominal Types
+
+Types that can be named directly. Specifically [enums], [structs], [unions],
+and [trait objects].
```

Comment:
Is it worth saying that "named" is "referred to by a path", or similar?

Perhaps also add Structural Types, and a cross-ref between them for contrast?

Reply:
Worth doing, but in its own PR.